gtest_parallel.py

```diff
@@ -44,6 +44,11 @@
   import fcntl
 
 
+# The log file names is truncated to 256 to overcome OS limitations
+MKSTEMP_RANDOM_LEN = 12  # length of mkstemp random string being added
+NAMEMAX_DEFAULT = 255  # Limit for log file name length
```

Comment:
Can this be renamed DEFAULT_PREFIX_LENGTH and maybe = 240?

Reply:
Sure, renamed and set to 240